src/foam/nanos/crunch/Capability.js

```diff
@@ -197,6 +197,13 @@ foam.CLASS({
       javaFactory: 'return foam.mlang.MLang.TRUE;',
       documentation: 'Predicate used to omit or include capabilities from capabilityDAO'
     },
+    {
+      class: 'foam.mlang.predicate.PredicateProperty',
+      name: 'visibilityPredicate',
```

Comment:
I'd like to name this something specific to capabilities so it's not confused with visibility in general. . 
```suggestion
      name: 'capabilityVisibilityPredicate',
```

Reply:
Actually looking at this, why is it needed? 
@nanoMichal I'm sure we have predicate to test for True already.

Reply:
@zahnno @Nauna

Reply:
So this came from availabilityPredicate.
So the requirement up to now was sufficient to use availabilityPredicate to filter the capabilityDAO for capability in the store.
However new requirement is to have the signingOfficer question capability visible and accessible in Ablii, but not visible but accessible in Treviso.
So now we have a difference in visibility and availability.

AvailabilityPredicate uses the CapabilityAvailabilityDAO and if predicate false returns no capability (null)
VisibilityPredicate will be 100% ui 

Although note this was a quick fix so we can move on with other work.

Phase 2 we have an idea to make a Cardlet - similar to Wizardlet, but for capability cards instead of wizard - and store UI components for Capability cards in the store.

Reply:
FYI - this was revision of @noodlemoodle 's original work. Ruby's work originally had this setup on availabilityPredicate and we were going to use permissions for dao availability. However setting up permissions was tedious at some points so this predicate (availabilityPredicate) was wanted to be as it was. Thus Ruby's work was reverted.
HOWEVER as described above we do need this logic ... thus new visibilityPredicate